### PR TITLE
Onboarding questions

### PR DESCRIPTION
**Pull request by NilsBacke**

### CODE REVIEW

**Comment by ArunJ98** [comment]
Some thoughts - It would be nice to have some validation for the inputs, for example, you can click through to next with an empty name. Other fields like graduation year could also be validated. Also, should the expected graduation field have a dropdown similar to selecting Academic year?

**Comment by NilsBacke** [comment]
> Some thoughts - It would be nice to have some validation for the inputs, for example, you can click through to next with an empty name. Other fields like graduation year could also be validated. Also, should the expected graduation field have a dropdown similar to selecting Academic year?

Great points! Validation is something I overlooked for sure. I was thinking expected graduation would just be a textfield because otherwise we would have to update it once a year to add the new freshman class graduation year. I'll put up a new commit with some changes shortly.

File: frontend/src/Onboarding/MajorScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        options={majors.map(maj => maj.name)}
+        renderInput={params => (
+          <TextField
+            {...params}
+            variant="outlined"
+            label="Select A Major"
+            fullWidth
+          />
+        )}
+        value={!!this.state.major ? this.state.major.name + " " : ""}
```
Comment (yasz24) [inline frontend/src/Onboarding/MajorScreen.tsx]: For some reason this doesn't seem to be working right. I don't see the major.name populated when it's selected from the drop-down

Comment (yasz24) [reply frontend/src/Onboarding/MajorScreen.tsx]: Also why the `!!` ?

Comment (NilsBacke) [reply frontend/src/Onboarding/MajorScreen.tsx]: This is really weird. It works fine for me when I test it locally using `yarn start`, but it doesn't work on zeit.

File: frontend/src/home/Home.tsx [hunk 2]
```diff
@@ -364,6 +362,7 @@ class HomeComponent extends React.Component<Props, HomeState> {
   }
 
   render() {
+    console.log("planStr " + this.state.planStr);
```
Comment (yasz24) [inline frontend/src/home/Home.tsx]: maybe get rid of this console.log? looks great otherwise

**Pull request merged**